extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -97,7 +96,7 @@ IgnorableNonIndexedClasses ignorable() {
         return new IgnorableNonIndexedClasses(ignorable);
     }
 
-    @BuildStep(onlyIf = NativeImageFutureDefault.CompleteReflectionTypes.class)
+    @BuildStep
```

Comment:
Ideally, if we decide to register it always, let's add it to the method below instead.

Reply:
OK.